Dockerfile

```diff
@@ -1,15 +1,15 @@
 # To build:
-# $ docker run --rm -v $(pwd):/go/src/github.com/micahhausler/es-proxy -w /go/src/github.com/micahhausler/es-proxy golang:1.7  go build -v -a -tags netgo -installsuffix netgo -ldflags '-w'
-# $ docker build -t micahhausler/es-proxy .
+# $ docker run --rm -v $(pwd):/go/src/github.com/skuid/es-proxy -w /go/src/github.com/skuid/es-proxy golang:1.7  go build -v -a -tags netgo -installsuffix netgo -ldflags '-w'
+# $ docker build -t skuid/es-proxy .
 #
 # To run:
-# $ docker run micahhausler/es-proxy
+# $ docker run skuid/es-proxy
 
 FROM alpine
 
 RUN apk -U add ca-certificates
 
-MAINTAINER Micah Hausler, <hausler.m@gmail.com>
+LABEL maintainer=devops@skuid.com
 
```

Comment:
Super minor, but could you add `EXPOSE` for 3000 and 3001?

Reply:
Ah, yes.